packages/rum/test/types/tsconfig.json

```diff
@@ -10,7 +10,8 @@
     "allowJs": true,
     "checkJs": true,
     "strict": true,
-    "esModuleInterop": true
+    "esModuleInterop": true,
+    "skipLibCheck": true,
```

Comment:
removed checking for typings from libraries which is not required for testing RUM types.